.travis-just-danger.yml

```diff
@@ -0,0 +1,10 @@
+language: node_js
+cache:
+  yarn: true
+  directories:
+  - node_modules
+
+node_js: node
+script:
+- yarn run link
+- danger run --verbose
```

Comment:
this isn't run by default, but it's nice when you're working on the dangerfile and want to test quickly